lib/ConvoyLeader/src/App.cpp

```diff
@@ -67,12 +74,34 @@ static const uint32_t SERIAL_BAUDRATE = 115200U;
 /** Serial log sink */
 static LogSinkPrinter gLogSinkSerial("Serial", &Serial);
 
+/* MQTT topic name for birth messages. */
+const char* App::TOPIC_NAME_BIRTH = "birth";
+
+/* MQTT topic name for will messages. */
+const char* App::TOPIC_NAME_WILL = "will";
+
+/* MQTT topic name for receiving position setpoint coordinates. */
+const char* App::TOPIC_NAME_POSITION_SETPOINT = "positionSetpoint";
+
+/** Default size of the JSON Document for parsing. */
+static const uint32_t JSON_DOC_DEFAULT_SIZE = 1024U;
+
+/** Buffer size for JSON serialization of birth / will message */
+static const uint32_t JSON_BIRTHMESSAGE_MAX_SIZE = 64U;
+
+/** HeadingFinder Instance. */
+static HeadingFinder gHeadingFinder;

```

Comment:
Why is it not a member variable?

Reply:
Because the callbacks of SerialMuxProt should be able to write the new data to it, which they cannot do if it is a member